DESCRIPTION

```diff
@@ -3,21 +3,21 @@ Title: Connect to ODBC Compatible Databases (using the DBI Interface)
 Version: 1.3.5.9000
 Authors@R: c(
     person("Jim", "Hester", role = "aut"),
-    person("Hadley", "Wickham", , "hadley@rstudio.com", role = c("aut", "cre")),
+    person("Hadley", "Wickham", , "hadley@posit.co", role = c("aut", "cre")),
     person("Oliver", "Gjoneski", role = "aut"),
     person("lexicalunit", role = "cph",
            comment = "nanodbc library"),
     person("Google Inc.", role = "cph",
            comment = "cctz library"),
-    person("RStudio", role = c("cph", "fnd"))
+    person("Posit Software, PBC", role = c("cph", "fnd"))
   )
 Description: A DBI-compatible interface to ODBC databases.
 License: MIT + file LICENSE
 URL: https://r-dbi.github.io/odbc/, https://github.com/r-dbi/odbc,
     https://db.rstudio.com
 BugReports: https://github.com/r-dbi/odbc/issues
 Depends: 
-    R (>= 3.2.0)
+    R (>= 3.6.0)
```

Comment:
Was initially hesitant here, but this seems right. 

The minimum version we test the package with (on `main`) is 3.6.0 / oldrel-4 (currently 3.6.x), released 4 years ago:

https://github.com/r-dbi/odbc/blob/505277d606a299d74c4c3abcdbc46345731694b5/.github/workflows/R-CMD-check.yaml#L29

https://github.com/r-dbi/odbc/blob/505277d606a299d74c4c3abcdbc46345731694b5/.github/workflows/R-CMD-check.yaml#L36

The R version was [most recently changed in 2019](https://github.com/r-dbi/odbc/blame/505277d606a299d74c4c3abcdbc46345731694b5/DESCRIPTION#L20), 4 years after the release of R 3.2.0.